documentation/TIPS.md

```diff
@@ -0,0 +1,38 @@
+# Welcome to OpenCircuits!
+
+I wanted to create this document to help you have a seamless transition into the team and answer some questions that you may have. Firstly, you may be wondering what OpenCircuits is. 
+OpenCircuits is an open-source circuit designer that was created by Leon Montealegre, who is the program manager. The team is also currently in the process of creating OpenCircuitsEE which is an open-source circuit designer geared towards electrical engineering and computer systems engineering students at RPI. We are hoping that OpenCircuitsEE will replace the current LTspice programs being used in the ECSE curriculum. You will have the freedom to work on either project throughout the semester.
+
+You should start off by navigating through the OpenCircuits website and familiarizing yourself with the tools (bonus points if you can find any bugs (http://opencircuits.io/).
+
+In OpenCircuits, you will be using GitHub which is a repository hosting service. If you are unfamiliar with GitHub, don’t be shy to ask Leon or other team members about how to navigate through Github! (https://github.com/OpenCircuits/OpenCircuits). 
+
+In order to get set up, make sure to access the OpenCircuits wiki page and follow the instructions to get set up (https://github.com/OpenCircuits/OpenCircuits/wiki).
+
+I would suggest downloading Visual Studio 2 to use are your Integrated Development Environment (IDE).
```

Comment:
Visual Studio Code also works well for me, I wouldn't push any specific text editor or IDE
> to use are your

*to use as your